### PR TITLE
OPS-5387 Enabled Spark notebooks to run in repo's Docker container.

### PR DESCRIPTION
**Pull request by neelp-git**

### CODE REVIEW

**Review by kirmatty26 — APPROVED** [review]

**Comment by arrowplum** [comment]
I have not run it but from a cursory glance LGTM

**Review by arrowplum — APPROVED** [review]

File: notebooks/spark/other_notebooks/AerospikeSparkPythonJSONSQL.ipynb [hunk 1]
```diff
@@ -5,66 +5,144 @@
    "metadata": {},
    "source": [
     "# Store JSON documents into Aerospike and query using Spark SQL\n",
-    "## Tested with Spark connector 3.1.0, Java 8, Apache Spark 3.0.2, Python 3.7"
+    "#### Tested with Spark connector 3.1.0, Java 8, Apache Spark 3.0.2, Python 3.7"
```
Comment (rahul-aerospike) [inline notebooks/spark/other_notebooks/AerospikeSparkPythonJSONSQL.ipynb:8]: Spark connector **3.2.0**

**Review by rahul-aerospike — CHANGES_REQUESTED** [review]
LGTM. Just a small change.

**Comment by neelp-git** [comment]
@rahul-aerospike Yes, we can claim tested with 3.2.0 because it does run successfully -thanks. Will update and merge.

**Pull request merged**